wordpress-mobile/WordPress-Android
# ProgressBar in Posts list

Adds an infinite (i.e. indeterminate) progress bar to each `PostsList ` item so we can show progress while media/posts are being uploaded in the background.

This lays the basis for a second step where we'll have a determinate progress bar in a separate PR.

To test:
1. go to the Posts list and start a new draft
2. include some media
3. exit the editor
4. the newly created Post should show the indeterminate progress bar while it uploads related media and the post itself, until it's fully uploaded and then such progress bar disappears.

Repeat the same with the Pages section.

cc @nbradbury
